CredentialsManager/FilesExceptions/DuplicateUserCredentialsException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace CredentialsManager.FilesExceptions
+{
+#pragma warning disable RCS1194 // Implement exception constructors.
+
+  public class DuplicateUserCredentialsException : Exception
+#pragma warning restore RCS1194 // Implement exception constructors.
+  {
+    public DuplicateUserCredentialsException()
```

Comment:
Get rid of this constructor (also, if you do, then you can probably get rid of that warning suppression).

Reply:
I updated it in [493cd86](https://github.com/csinn/CSharp-From-Zero-To-Hero-v2/pull/43/commits/493cd86aa678a20b0c535f9fbea6a18198e61285)